### ISSUE TITLE
Accessibility Bug: This nav does not have a unique accessible name as required when two or more nav elements are present.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3155]
Fundamental core component: "Pagination"

Description: This nav does not have a unique accessible name as required when two or more nav elements are present.

![image](https://user-images.githubusercontent.com/32538291/91651833-17eff680-eaaf-11ea-9801-bb3765d4e66a.png)

Expected: When there is ambiguity about an ARIA region, landmark or HTML section, developers must ensure that these elements contain meaningful accessible labels to assist users of screen readers to identify and distinguish the section, region or landmark

Screenshot 1:

![image](https://user-images.githubusercontent.com/32538291/91651840-348c2e80-eaaf-11ea-95d4-a43d0a77c882.png)

Screenshot 2:

![image](https://user-images.githubusercontent.com/32538291/91651846-3bb33c80-eaaf-11ea-9155-13e967b98825.png)

**Comment by kavya-b** [comment]
This seems to be fixed with https://github.com/SAP/fundamental-ngx/pull/5460.
@Sanjay-1312 @puru-hk please verify with latest netlify https://fundamental-ngx.netlify.app/#/core/pagination

**Comment by Sanjay-1312** [comment]
![image](https://user-images.githubusercontent.com/78714518/123457934-12493b80-d602-11eb-885f-067877d5308a.png)
i can still able to reproduce the issue.

URL: https://fundamental-ngx.netlify.app/#/core/pagination
